src/common/tokenHelpers.ts

```diff
@@ -0,0 +1,40 @@
+import { asObject, asString } from 'cleaners'
+import { EdgeToken, EdgeTokenMap } from 'edge-core-js/types'
+
+/**
+ * The `networkLocation` field is untyped,
+ * but many currency plugins will put a contract address in there.
+ */
+export const asContractLocation = asObject({
+  contractAddress: asString
+})
+
+export function getEosTokenId(token: EdgeToken): string {
```

Comment:
This doesn't feel clean to have asset specific logic in a `common` file. I would prefer to move these to asset specific files. Besides, they are only used by those specific asset files.

Reply:
I deleted all of them, and just left the common cleaner.